src/app/static/src/app/store/modelRun/mutations.ts

```diff
@@ -51,6 +51,7 @@ export const mutations: MutationTree<ModelRunState> = {
     },
 
     [ModelRunMutation.RunStatusError](state: ModelRunState, action: PayloadWithType<Error>) {
+        state.pollingCounter += 1
```

Comment:
Of course, incrementing the poll here means you'll hit the limit one iteration earlier. I guess that's ok at this sort of scale.